snorkel/slicing/utils.py

```diff
@@ -128,7 +130,9 @@ def convert_to_slice_tasks(base_task: Task, slice_names: List[str]) -> List[Task
         ind_head_op = Operation(
             module_name=ind_head_module_name, inputs=head_module_op.inputs
         )
-        ind_task_flow = body_flow + [ind_head_op]
+        ind_task_ops = [ind_head_op]
+        ind_task_flow = body_flow + ind_task_ops
+        slice_task_ops.extend(ind_task_ops)
```

Comment:
nit: could just extend() then append()

Reply:
sorry I'm not following—extend which?

Reply:
```python
slice_task_ops.extend(body_flow)
slice_task_ops.append(ind_head_op)
```

a bit clearer to me

Reply:
Can't do that because we need `task_flow` to define the `Task`.
I cleaned up the comments a bit to segment the steps a bit more: 
```
        # Define operations for task head
        pred_transform_op = Operation(
            module_name=pred_transform_module_name, inputs=head_module_op.inputs
        )
        pred_head_op = Operation(
            module_name=pred_head_module_name, inputs=[(pred_transform_op.name, 0)]
        )
        pred_task_ops = [pred_transform_op, pred_head_op]
        slice_task_ops.extend(pred_task_ops)

        # Create task flow
        pred_task_flow = body_flow + pred_task_ops

        # Create pred task
```